index.Rmd

```diff
@@ -8,14 +8,14 @@ output:
     df_print: paged
 ---
 
-# Introduction
+# Introduction 
 Opioid analgesics are pain relievers derived from opium or have an opium-like activity. There are no better drugs than opioids for treating severe pain and suffering, however, opioids are the main drugs associated with overdose deaths. Opioid prescription rates have increased almost threefold associated with an increase of opioid related overdoses and deaths in the last 15 years. New York has been greatly impacted by the opioid epidemic. The rate of deaths related to any opioid in New York has increased by 210% from 2010 to 2016. The opioid overdose death rate in the overall state was 18 deaths per 100,000 residents, which was higher than many states in the United States.  
 
 # Materials and methods
 
 
-1. Download all packages 
-2. Downlaod opioid data and New York census, and join opioid data and New York census
```

Comment:
Please expand this to be a narrative instead of bullet points. Where did the data come from?  how can someone find them?